addons/hr_skills/__manifest__.py

```diff
@@ -36,11 +36,12 @@
         'data/hr.resume.line.csv',
     ],
     'installable': True,
+    'auto_install': True,
     'application': True,
     'assets': {
         'web.assets_backend': [
-            'hr_skills/static/src/fields/skills_one2many.xml',
-            'hr_skills/static/src/fields/*',
+            'hr_skills/static/src/fields/skills_one2many/*',
```

Comment:
any reason why you need skills_one2many to be loaded before the other fields?

Reply:
A field inherits a template from another and doesn't work if i don't load this one first